docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs

```diff
@@ -49,21 +59,28 @@ public static void Example()
                     Console.WriteLine("");
                 }
 
-                Console.WriteLine("===================================================");
+                Console.WriteLine(
+                    "===================================================");
```

Comment:
this looks ugly, just shorten the output a bit

Reply:
Either we do it for all files or we just go with the tool restricting line width to 85 characters :)

Reply:
so why do we need to shorten to 85 chars? what are the benefits? 

---
In reply to: [278267763](https://github.com/dotnet/machinelearning/pull/3561#discussion_r278267763) [](ancestors = 278267763)

Reply:
@ganik Its a requirement from the docs team. @natke can explain more.

Reply:
Don't blame the messenger! If you includes samples that are less than 85 chars, then readers do not have to scroll left and right to read them.